dts/bindings/serial/nxp,lpc-usart.yaml

```diff
@@ -5,4 +5,12 @@ description: LPC USART
 
 compatible: "nxp,lpc-usart"
 
+properties:
```

Comment:
It would be good to put this in `dts/bindings/serial/uart-controller.yaml ` and remove it from dts/bindings/serial/st,stm32*.yaml as well.  Also can we use an `enum` for the values.

Reply:
@galak, I'm totally agree with you, see if changes is okay for you.
Also, may I should rename the PR ?